src/current/cockroachcloud/cloud-api.md

```diff
@@ -380,10 +420,66 @@ Where:
 
   - `{cluster_id}` is the unique ID of this cluster.
   {{site.data.alerts.callout_info}}
-  The cluster ID used in the Cloud API is different from the routing ID used when [connecting to clusters]({% link cockroachcloud/connect-to-a-basic-cluster.md %}).
+  The cluster ID used in the Cloud API is different from the routing ID used when [connecting to clusters]({% link cockroachcloud/connect-to-your-cluster.md %}).
   {{site.data.alerts.end}}
   - `{secret_key}` is the secret key for the service account.
-  - `{spend_limit}` is the [maximum amount of money, in US cents, you want to spend per month]({% link cockroachcloud/plan-your-cluster.md %}) on this cluster.
+  - `storage_mib_limit` is the maximum number of Mebibytes of storage that the cluster can have at any time during the month. If this limit is exceeded, then the cluster is throttled, where only one SQL connection is allowed at a time, with the expectation that it is used to delete data to reduce storage usage. It is an error for this value to be zero.
+  - `request_unit_limit` is the maximum number of request units that the cluster can consume during the month. If this limit is exceeded, then the cluster is disabled until the limit is increased, or until the beginning of the next month when more request units are granted. It is an error for this to be zero.
+
+If the request was successful, the client will not receive a response payload.
```

Comment:
```suggestion
If the request is successful, the command returns no response.
```

This phrasing occurs several times in the file.

Reply:
`command returns no response` is technically incorrect. How about we reference the response code here?   For example, "a successful response will return a 200 code despite the response body being empty" or something like that.  Also, is this for a PATCH on `https://cockroachlabs.cloud/api/v1/clusters/{cluster_id}` ?  That should return a body.  Does it not?